### DISCUSSION TITLE
Save user assignment with start and end date, and keep history

### DISCUSSION THREAD

**Discussion by rhahao** [sws2apps/organized-app#17] [Ideas]
It would be good if the students assignment would have start and end date, and history can be kept inside the apps

**Comment by rhahao** [comment]
This feature has been shipped starting with [LMM-OA v1.0.0-alpha.137](https://github.com/sws2apps/lmm-oa-sws/releases/tag/v1.0.0-alpha.137).

![assignments_history](https://user-images.githubusercontent.com/26148770/178285940-53bf16b0-6849-42f5-9249-fd7eeb98f1fc.png)

**Reply by joaochris** [reply]
Do the comments show up somewhere, or just on this screen? And what does that start and end date specifically refer to?

**Comment by rhahao** [comment]
Hello @joaochris , yes, the comments only appear here. (at least for now). These dates indicate when a person will be available for an assignment and when it ends (if for some reason, they are no longer qualified)

**Reply by joaochris** [reply]
And the time of absence? I got a little confused 😂

**Reply by rhahao** [reply]
The time of absence also works the same. The LMM Overseer can enter the person time away here, and during those period, the person will not be available for any assignment.
